test/integration/backward_compatible/LoggingTest.js

```diff
@@ -69,15 +71,15 @@ describe('LoggingTest', function () {
     });
 
     beforeEach(function () {
-        sinon.spy(console, 'log');
+        consoleLogSpy = sinon.spy(console, 'log');
     });
 
     afterEach(async function () {
         if (client != null) {
             await client.shutdown();
             client = null;
         }
-        console.log.restore();
+        consoleLogSpy.restore();
```

Comment:
this reporter changes how console.log works, so we need to use variable here for this test to pass